api/shorelineApi.go

```diff
@@ -23,8 +23,10 @@ type (
 		ServerSecret         string `json:"serverSecret"` //used for services
 		LongTermKey          string `json:"longTermKey"`
 		LongTermDaysDuration int    `json:"longTermDaysDuration"`
-		Salt                 string `json:"salt"`      //used for pw
-		Secret               string `json:"apiSecret"` //used for token
+		Salt                 string `json:"salt"`                //used for pw
+		Secret               string `json:"apiSecret"`           //used for token
+		VerificationSkipped  bool   `json:"verificationSkipped"` //allows for the skipping of verification for testing
```

Comment:
I don't think that VerificationSkipped (past tense) is the right name for this. VerificationSkippable, perhaps, or maybe CanSkipVerification would be better.

Or perhaps you could eliminate this variable entirely and just say that if VerificationSecret is either missing or an empty string, you *cannot* skip verification.